src/components/Help/Help.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
+import { Text } from 'ink';
+import { HelpMessage } from "../../CliModels";
+
+interface HelpInput {
+	message: HelpMessage,
+}
+
+const Help: FC<HelpInput> = ({ message }) => (
+	<Text>{message}</Text>
+);
+
+module.exports = Help;
+export default Help;
```

Comment:
Why do you use CJS and ESM exports? If you use TS and your output format will be CJS, then TS will transform to CJS, so use only `export default Help` :)

Reply:
I copy paste from the generated components by Ink boilerplate, I will delete the CJS ones! Thanks!